virtual_ecosystem/models/soil/pools.py

```diff
@@ -755,3 +876,111 @@ def calculate_soil_nutrient_mineralisation(
 
     carbon_nutrient_ratio = pool_carbon / pool_nutrient
     return breakdown_rate / carbon_nutrient_ratio
+
+
+def calculate_nutrient_flows_to_necromass(
+    microbial_changes: MicrobialChanges, constants: SoilConsts
+) -> NDArray[np.float32]:
+    """Calculate the rate at which nutrients flow into the necromass pool.
+
+    These flows comprise of the nitrogen and phosphorus content of the dead cells and
+    denatured enzymes that flow into the necromass pool.
+
+    TODO - A core assumption here is that the stochiometry of the enzymes are identical
+    to the microbial cells. This assumption works for now but will have to be revisited
+    when fungi are added (as they have different stochiometric ratios but will
+    contribute to the same enzyme pools)
```

Comment:
I guess you have a separate function for this and the chosen inputs as there might be other, more complex stuff going on in the future in this function, right? Otherwise, this functions seems a bit overkilling for a simple division.

Reply:
Yep in future, the microbial pool will be separated into (at least) a bacterial pool and a fungal pool. These pools will have different stoichiometry which means the calculation will become more involved, so the function is basically a placeholder for when I will do this.